src/pd/util.rs

```diff
@@ -365,7 +365,7 @@ pub fn validate_endpoints(
             Ok(resp) => resp,
             // Ignore failed PD node.
             Err(e) => {
-                error!("PD failed to respond"; "endpoints" => ep, "err" => ?e);
+                debug!("PD failed to respond"; "endpoints" => ep, "err" => ?e);
```

Comment:
I prefer `warn`.

Reply:
@overvenus this message is every 300ms.  At warn the log file will start to fill up.  There is a warn every 10 attempts.

Reply:
Currently, we have 2 WARN logs that happen every time.

Should it be a WARN log every so many attempts, and `debug` for each attempt? Or maybe `info`?

Reply:
Did not notice that there are other warns, could you change it to `info`? So we can know when it fails.